frappe/public/js/frappe/form/grid_row.js

```diff
@@ -229,7 +229,7 @@ export default class GridRow {
 				this.open_form_button = $(`
 					<div class="btn-open-row">
 						<a>${frappe.utils.icon('edit', 'xs')}</a>
-						<div class="hidden-xs edit-grid-row">Edit</div>
+						<div class="hidden-xs edit-grid-row">` + __("Edit") + `</div>
```

Comment:
```suggestion
						<div class="hidden-xs edit-grid-row">${ __("Edit") }</div>
```

Like in the line above, the translate function can be called within `${ }`. See https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/template_strings and https://discuss.erpnext.com/t/dos-and-don-ts-2-translation-in-javascript/68505